### ISSUE TITLE
[Bug] Signed integer is greater than maximum

### ISSUE THREAD

**Issue by Subtixx** [rommapp/romm#1368]
**RomM version**
rommapp/romm:latest@sha256:cc7afe6027734742d70dd2c1419929df45e1811226a1b089fe8e9b67d349bb93

**Describe the bug**
Scanning platform PS2 fails with an error

**To Reproduce**

1. I don't know. It happens everytime I scan my PS2 folder.

**Expected behavior**
Scan successful

**Desktop (please complete the following information):**

(I dont think this is relevant)

- OS: Arch Linux
- Browser Firefox
- Version 133.0.3 (64-bit)

**Additional context**
```
CallbackIO: write. filename=Action Man A.T.O.M. - Alpha Teens on Machines (Europe) (En,Fr,De,Es,It,Nl,No,Da,Fi).chd
ERROR:    [RomM][scan][2024-12-19 07:45:27] signed integer is greater than maximum
```

**Comment by zurdi15** [comment]
Does it always happens with the same game? Can you try to remove it temporarily from your library and scan again?

**Comment by Subtixx** [comment]
Yes it always happens with the same game, removing the game results in scanning to work sucessfully

I even tried ripping the game again but with the new rip (same hash) same problem

**Comment by gantoine** [comment]
Well I found the game as at 7zip, extracted as a folder and transformed into CHD and they all scan fine. You might have a different copy, in which case I'd suggest dumping another copy of the game.

**Comment by Subtixx** [comment]
I did dump it myself (twice)

**Comment by gantoine** [comment]
This is a very specific issue, join the discord and post something in `#open-development` so we can keep looking into this directly. https://discord.gg/8p9993wk
